examples/cmd/webapp-influxdb/main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"sourcegraph.com/sourcegraph/appdash"
+	"sourcegraph.com/sourcegraph/appdash/httptrace"
+	"sourcegraph.com/sourcegraph/appdash/traceapp"
+
+	"github.com/codegangsta/negroni"
+	"github.com/gorilla/context"
+	"github.com/gorilla/mux"
+
+	influxDBServer "github.com/influxdata/influxdb/cmd/influxd/run"
+)
+
+const CtxSpanID = 0
+
+var collector appdash.Collector
+
+func main() {
+	conf, err := influxDBServer.NewDemoConfig()
+	if err != nil {
+		log.Fatalf("failed to create influxdb config, error: %v", err)
+	}
+
+	// Enables InfluxDB server authentication.
+	conf.HTTPD.AuthEnabled = true
+
+	// InfluxDB server auth credentials.
+	// If user does not exist yet it will
+	// created as admin user.
```

Comment:
replace with (80 character width, typo "s/it will created/it will be created/g"):

```
// InfluxDB server auth credentials. If user does not exist yet it will be
// created as an admin user.
```